packages-exp/auth-exp/src/core/user/token_manager.ts

```diff
@@ -76,6 +69,17 @@ export class StsTokenManager {
   async getToken(
     auth: AuthInternal,
     forceRefresh = false
+  ): Promise<string | null> {
+    if (!this.isPassthroughMode) {
```

Comment:
remove the `!` and switch the order of the if/else

Reply:
Done!